tests/lints/check-used-functions.py

```diff
@@ -35,6 +35,15 @@
     r"(?P<name>\b(std)?(::)?strtod\b)": "parse<double>",
     r"(?P<name>\b(std)?(::)?strtold\b)": "parse<double>",
     r"(?P<name>\b(std)?(::)?sscanf\b)": "scan",
+    r"(?P<name>\b(std)?(::)?isalnum\b)": "is_ascii_TODO",
```

Comment:
Is this used?

Reply:
whoops, good catch. This is not yet used, I wanted to add it for completeness. I'll change this!